### DISCUSSION TITLE
Theming with SASS

### DISCUSSION THREAD

**Discussion by gempain** [mantinedev/mantine#169] [Q&A] [answered]
Hi 👋 
The [docs](https://mantine.dev/theming/extend-theme/) show that theming is done with CSS-in-JS. Is there a way to style components with SASS modules ?
Keep up the great work !

**Comment by rtivital** [comment, marked as answer]
Hi, Mantine i built with react-jss, we do not use sass and cannot provide such option

**Reply by gempain** [reply]
This is unfortunate, but thanks for the response.

**Reply by zac18992** [reply]
Hi @rtivital is this still the case? I read that the library will be moving over to css or sass?

Next.js supports css/sass modules out of the box and it renders the styles on the server render (even if JS is disabled). For example, just run `npx create-next-app@latest`, run `yarn add sass` in the project, and add some and import some component styles, eg `import styles from './Component.module.scss'` file.

However, if I add scss modules in exactly the same way to [Mantine+Next](https://github.com/mantinedev/mantine-next-template), the sass styles do not load on the server render, only on the client.

Do you know why Mantine breaks the sass/css modules or how I can integrate them?
